api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -27,7 +27,7 @@ import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service ConfigMgmt {
   /*
-  GetNodes
+  List Checked-in Nodes
```

Comment:
wait i thought we were updating these to show?

Reply:
Those that have an authorization action of "list" were changed to "list" (see line 45).
Those that have an authorization action of "get" were changed to "show" (see line 160).